### PR TITLE
remove static modifier #674

### PR DESCRIPTION
**Pull request by btkobayashirun**
Please review #674.

● `ErrorController.java `
　with an RequestMethod type(get or post) that calls each method.

● `LoginController.java `
　Because post requests are generated by [forwarding](https://github.com/terasolunaorg/terasoluna-tourreservation/blob/master/terasoluna-tourreservation-web/src/main/resources/META-INF/spring/spring-security.xml#L40), both Get and POST are accepted.
　This fix does not resolve SonarQube's point.

● `ReservationReportPdfStamperView.java`
　The policy of the modification is as follows.
　https://rules.sonarsource.com/java/tag/multi-threading/RSPEC-2885  
 
　The name of the variable is the same as [ManageReservationHelper](https://github.com/terasolunaorg/terasoluna-tourreservation/blob/1862c937050faaa07837a68d4e3a2878099b2b17/terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ManageReservationHelper.java#L160).

### CODE REVIEW

File: terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java [hunk 1]
```diff
@@ -32,14 +32,14 @@
...
 
-    private static final SimpleDateFormat SDF = new SimpleDateFormat("yyyy/MM/dd");
-
     private static final String DOWNLOAD_PDF_EXTENSION = ".pdf";
 
     private static final int REFERENCE_NAME_MAXIMUM_VALUE_WITH_NORMAL_FONTSIZE = 30;
 
     private static final float REFERENCE_NAME_VARIABLE_FONTSIZE = 8.0F;
 
+    private final SimpleDateFormat sdf = new SimpleDateFormat("yyyy/MM/dd");
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java:41]: - [ ] I essentially recommend using JSR-310 to format with DateTimeFormatter. (Not a comment that mandatory to fix)

File: terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/error/ErrorController.java [hunk 2]
```diff
@@ -26,7 +28,7 @@
      * Shows the access denied error view.
      * @return access denied error view
      */
-    @RequestMapping("accessDeniedError")
+    @GetMapping("accessDeniedError")
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/error/ErrorController.java:31]: - [ ] You should consider replacing any RequestMapping attached to the handler method, or not.
I don't recommend implementing it with a partially different policy, just to solve the SonarQube issue.

File: terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java (hunk 1 above)
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java:41]: > The name of the variable is the same as ManageReservationHelper.

- [ ] In essence, you should have a SimpleDateFormat as a bean variable with the format string specified in the property. (Not a comment that mandatory to fix)

File: terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/error/ErrorController.java (hunk 2 above)
Comment (btkobayashirun) [reply terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/error/ErrorController.java]: Return to RequestMapping according to the original policy.

d090604be30bf902102e54b48d0e236ca9ac71fd

File: terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java (hunk 1 above)
Comment (btkobayashirun) [reply terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java:41]: We have created another Issue https://github.com/terasolunaorg/terasoluna-tourreservation/issues/676

Comment (btkobayashirun) [reply terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java:41]: We have created another Issue https://github.com/terasolunaorg/terasoluna-tourreservation/issues/676

**Pull request merged**